src/main/java/de/hysky/skyblocker/skyblock/item/CustomArmorTrims.java

```diff
@@ -137,6 +140,11 @@ private static int customizeTrim(FabricClientCommandSource source, Identifier ma
 	}
 
 	public record ArmorTrimId(@SerialEntry Identifier material, @SerialEntry Identifier pattern) implements Pair<Identifier, Identifier> {
+		public static final Codec<ArmorTrimId> CODEC = RecordCodecBuilder.create(instance -> instance.group(
+				Identifier.CODEC.fieldOf("material").forGetter(ArmorTrimId::material),
+				Identifier.CODEC.fieldOf("pattern").forGetter(ArmorTrimId::pattern)
+				).apply(instance, ArmorTrimId::new));
```

Comment:
Formatting error I think?

Reply:
I moved the bracket because I think its better on the line, the rest is intentional for readability